dvc/command/repro.py

```diff
@@ -51,7 +51,7 @@ def add_arguments(repro_parser):
         "targets",
         nargs="*",
         help=f"Stages to reproduce. '{PIPELINE_FILE}' by default.",
-    ).complete = completion.DVC_FILE
+    ).complete = completion.DVCFILES_AND_STAGE
```

Comment:
Adding this only to the `repro` for now. The performance is really bad, considering the reason I would do a completion for is to be faster.

Reply:
The performance isn't _that_ bad. 😄  At least in our example-get-started repo.